src/main/java/io/camunda/zeebe/bpmnassert/testengine/ZeebeEngine.java

```diff
@@ -0,0 +1,20 @@
+package io.camunda.zeebe.bpmnassert.testengine;
+
+import io.camunda.zeebe.client.ZeebeClient;
+
+public interface ZeebeEngine extends RecordStreamSource {
+
+  void start();
+
+  void stop();
+
+  ZeebeClient createClient();
+
+  String getGatewayAddress();
+
+  ZeebeEngineClock clock();
```

Comment:
:wrench: refactoring proposal. Instead of returning the actor clock, we could just offer the method `increaseTime()`here. This way we could get rid of `ZeebeEngineClock` and `ZeebeEngineClockImpl`